src/utils/runEndTime.ts

```diff
@@ -1,4 +1,6 @@
 export function runEndTime(runStart: string, runDuration: number): Date {
   const runStartTime = new Date(runStart)
-  return new Date(runStartTime.getTime() + (runDuration / 1000000))
+  if (!runDuration) runDuration = 0
+  const d =  new Date(runStartTime.getTime() + (runDuration / 1000000))
+  return d
```

Comment:
Maybe just `return new Date(runStartTime.getTime() + (runDuration / 1000000))`

Reply:
ahh yes, ty. Hold over from when I was testing & needed log the output